src/js/components/Booking.js

```diff
@@ -229,7 +229,8 @@ class Booking{
       body: JSON.stringify(payload),
     };
 
-    fetch(url, options);
+    fetch(url, options)
+    .then(thisBooking.booked);
```

Comment:
chodziło o to by zapisać udaną rezerwację a więc:
fetch(url, options).then((response) => {
// 1. np sprawdzenie if (response.ok) {
    // 2. dodanie rezerwacji (czyli chyba metoda `makeBooked` i przekazanie do niej odpowiednich wartosci
    // 3. wyczyszczenie wszystkich zmiennych (pick a time, table, people, hours, starters, inputy itp) do wartosci poczatkowych
    }
});